keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -287,6 +278,23 @@ open class SqlResourceRepository(
     }
   }
 
+  override fun <T : ResourceEvent> lastEventByType(id: String, eventType: Class<T>): T? {
+    return sqlRetry.withRetry(READ) {
+      jooq
+        .select(EVENT.JSON)
+        .from(EVENT, RESOURCE)
+        .where(EVENT.SCOPE.eq(Scope.RESOURCE.name))
+        .and(RESOURCE.ID.eq(id))
+        .and(RESOURCE.UID.eq(EVENT.UID))
```

Comment:
This line looks like an accident to me (i.e. remove entirely instead fixing the `eq` condition)

Reply:
This is the join condition between the two tables?